core/indexer/common.go

```diff
@@ -198,7 +198,7 @@ func (cm *commonProcessor) convertScResultInDatabaseScr(sc *smartContractResult.
 		Code:          string(sc.Code),
 		Data:          decodedData,
 		PreTxHash:     hex.EncodeToString(sc.PrevTxHash),
-		CallType:      string(sc.CallType),
+		CallType:      string(rune(sc.CallType)),
```

Comment:
Why do we cast to rune?

Reply:
because for the moment call type is a strange character. also golang suggestion to use rune

Reply:
isn't callType integer (0, 1, 2)? if you save it as string (utf8) of course callType will be a strange character. You should save it like this I think :) `strconv.Itoa(sc.CallType)`, this way you will get the correct value into the database (or better save it as integer in the json)

Reply:
refactored.